src/gcn.jl

```diff
@@ -102,6 +103,33 @@ function lower_throw_extra!(mod::LLVM.Module)
     end
     return changed
 end
+function fix_alloca_addrspace!(fn::LLVM.Function)
+    changed = false
+    alloca_as = 5
+
+    for bb in blocks(fn)
+        for insn in instructions(bb)
+            if isa(insn, LLVM.AllocaInst)
+                ty = llvmtype(insn)
+                ety = eltype(ty)
+                addrspace(ty) == alloca_as && continue
+
+                new_insn = nothing
+                let builder = Builder(JuliaContext())
```

Comment:
See https://github.com/JuliaGPU/GPUCompiler.jl/pull/70, you can't use `JuliaContext()` like that anymore. Here, you can derive the context, e.g. with `ctx = context(fn)` at the start of this function.